multimodal/src/autogluon/multimodal/utils/onnx.py

```diff
@@ -7,13 +8,16 @@
 
 logger = logging.getLogger(__name__)
 
-try:
-    import tensorrt  # Unused but required by TensorrtExecutionProvider
-except:
-    logger.warning(
-        "Failed to import tensorrt package. "
-        "onnxruntime would fallback to CUDAExecutionProvider instead of using TensorrtExecutionProvider."
-    )
+# TODO: Try a better workaround to lazy import tensorrt package.
+tensorrt_imported = False
```

Comment:
Seems removable?

Reply:
Yes, this might related to the import sequence with onnruntime package. But's it's hard to ensure onnxruntime wouldn't be imported beforehand.